src/astar/rocstar.js

```diff
@@ -142,8 +138,7 @@ const main = async () => {
 
     if (proxyBalance.free.lt(minBalance)) {
         console.log(`\nTopping up the proxy account on Shibuya with ${symbol} ...\n`);
-        const amount = new BN(100, 10);
-        const amountBN = amount.mul(decimalBN);
+        const amountBN = minBalance.muln(2);
```

Comment:
What is this muln() function? I don’t see it from the bn.js library.

Reply:
https://github.com/indutny/bn.js/#postfixes

2 is a number.
`minBalance.muln(2)` is equal to `minBalance.mul(new BN(2))`

Reply:
Okay ... the thinking behind mulplying it by 2 is confusing and makes the code hard to read. If minBalance is the minimum requirement, why do we need to multiply it? Is `2` a special number? If you worry the balance is not enough, why not multiply by `10`?